libs/cart/src/reducers/cart-shipping-information/cart-shipping-information.reducer.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  DaffCartShippingInformationActionTypes,
+} from '../../actions';
+import { initialState } from '../cart-initial-state';
+import { CartState } from '../cart-state.interface';
+import { ActionTypes } from '../action-types.type';
+
+export function reducer(
+  state = initialState,
+  action: ActionTypes
+): CartState {
+  switch (action.type) {
+    case DaffCartShippingInformationActionTypes.CartShippingInformationLoadAction:
+    case DaffCartShippingInformationActionTypes.CartShippingInformationUpdateAction:
+    case DaffCartShippingInformationActionTypes.CartShippingInformationDeleteAction:
+      return { ...state, loading: true };
+
+    case DaffCartShippingInformationActionTypes.CartShippingInformationLoadSuccessAction:
+      return {
+        ...state,
+        cart: {
+          ...state.cart,
+          // TODO: remove workaround
+          shipping_information: {...action.payload, address_id: null}
```

Comment:
I suppose the comment is explaining the `address_id: null`. But why is this here?

Reply:
Becuase the shipping information driver returns a `DaffCartShippingRate` not a `DaffCartShippingInformation`. I left a comment in slack suggesting that we change the driver to return the shipping info as it would eliminate workarounds like this and multiple locations in tests.

Reply:
Ok. I don't understand fully what's going on here but I added it to the list of things to take care of later.